_posts/2024-12-27-quasar-mejora-facturas.md

```
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Nuevos métodos de análisis de facturas: Mercado de Facturas y Facturas por ejecutivo.
+subtitle: Matriz de Riesgo - Cliente.
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [cliente, comercial, quasar, datos, facturas]
+comments: false
+author: Erick Burgos
+---
+
+## Mercado de Facturas
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)